urllib3/poolmanager.py

```diff
@@ -142,7 +137,7 @@ def connection_from_url(self, url):
         u = parse_url(url)
         return self.connection_from_host(u.host, port=u.port, scheme=u.scheme)
 
-    def urlopen(self, method, url, redirect=True, **kw):
+    def urlopen(self, method, url, redirect=True, retries=None, **kw):
```

Comment:
I can't see why `retries` needed to become an explicit parameter here. Care to explain the reasoning?